packages/@webex/plugin-meetings/src/roap/turnDiscovery.ts

```diff
@@ -274,12 +274,21 @@ export default class TurnDiscovery {
    * so it works fine no matter if TURN discovery is done or not.
    *
    * @param {Meeting} meeting
-   * @param {Boolean} isReconnecting should be set to true if this is a new
+   * @param {Boolean} [isReconnecting] should be set to true if this is a new
    *                                 media connection just after a reconnection
+   * @param {Boolean} [ignoreReachabilityResults]
    * @returns {Promise}
    */
-  async doTurnDiscovery(meeting: Meeting, isReconnecting?: boolean) {
-    const turnDiscoverySkippedReason = await this.getSkipReason(meeting);
+  async doTurnDiscovery(
+    meeting: Meeting,
+    isReconnecting?: boolean,
+    ignoreReachabilityResults?: boolean
```

Comment:
with the way this is implemented in this PR, when `ignoreReachabilityResults` is true, `doTurnDiscovery()` also ignores the config, so maybe a better name for this argument would be "isForced" or something like that, something to indicate that it will do the turn discovery no matter what

Reply:
Changed to ```isForced```, also removed the config check for enableTurnDiscovery everywhere since it was GA-ed a year ago